src/adapters/givPowerBalanceAggregator/givPowerBalanceAggregatorInterface.ts

```diff
@@ -0,0 +1,48 @@
+export interface GetLeastIndexedBlockTimeStampInputParams {
+  networks?: string; // comma separated sample: 100,420
+  network?: string | number; // comma separated sample: 100
```

Comment:
It's not comma separated 
I would say let's skip `network` and just use `networks`